python/ray/_private/worker.py

```diff
@@ -43,7 +43,7 @@
     from typing_extensions import Literal, Protocol
 
 import ray
-import ray._private.import_thread as import_thread
+import ray._private.gcs_utils as gcs_utils
```

Comment:
This is not allowed (there is a check that will fail) as it brings grpc back into the `import ray` critical path :)

But you don't need this, right?